dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -100,6 +100,6 @@
       "level_#{level.id}",
       #{standalone},
       "#{data['type']}" == "multi2" ? 2 : 1,
-      #{data['answers'].map {|answer| answer['correct']}},
+      #{data['answers'].map {|answer| answer['correct'] || 'null'}},
```

Comment:
It looks like it got checked in like this, which seems to be the cause of the break in multis, since 'null' is considered true by validateAnswers.